infrastructure-playbooks/purge-cluster.yml

```diff
@@ -343,6 +343,10 @@
     failed_when: false
     register: ceph_journal_partlabels
 
+  - name: see if encrypted partitions are present
+    command: blkid -t TYPE=crypto_LUKS -o value -s PARTUUID
```

Comment:
but this will get you every encrypted partition in the system. Isn't there a way to filter out what belongs to the OSD?